src/ngraph/runtime/cpu/mkldnn_emitter.hpp

```diff
@@ -277,6 +277,7 @@ namespace ngraph
 
                     mkldnn::post_ops ops;
 
+#if 0
```

Comment:
an `#if 0` here. maybe we can get rid of the code there, unless it serves some purpose?